### PR TITLE
PLAT-11296 Generator 2.0

### PR DESCRIPTION
**Pull request by symphony-thibault**
Preparing the 2.0 version of the `symphony-generator` that get rid of legacy SDKs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-# Symphony BDK Generator
+# Symphony BDK2.0 Generator
```
Comment (symphony-youri) [inline README.md:1]: I would not even mention 2.0 here

File: package.json [hunk 2]
```diff
@@ -23,10 +23,11 @@
     "node-openssl-p12": "^0.1.8",
     "selfsigned": "^1.10.8",
     "upath": "^1.1.2",
-    "yeoman-generator": "^4.0.2"
+    "yeoman-generator": "^4.13.0"
```
Comment (symphony-youri) [inline package.json:26]: should we try to migrate to version 5?

File: yarn.lock [hunk 3]
```diff
@@ -127,10 +120,10 @@
   dependencies:
     "@babel/types" "^7.11.0"
 
-"@babel/helper-validator-identifier@^7.10.4":
-  version "7.10.4"
-  resolved "https://registry.npmjs.org/@babel/helper-validator-identifier/-/helper-validator-identifier-7.10.4.tgz"
-  integrity sha512-3U9y+43hz7ZM+rzG24Qe2mufW5KhvFg/NhnNph+i9mgCtdTCtMJuI1TMkrIUiK7Ix4PYlRF9I5dhqaLYA/ADXw==
+"@babel/helper-validator-identifier@^7.10.4", "@babel/helper-validator-identifier@^7.14.5":
+  version "7.14.9"
+  resolved "https://repo.symphony.com/artifactory/api/npm/npm-virtual-dev/@babel/helper-validator-identifier/-/helper-validator-identifier-7.14.9.tgz"
```
[the remark this answers is not in the document]
Comment (symphony-thibault) [reply yarn.lock]: Actually reverted through https://github.com/SymphonyPlatformSolutions/generator-symphony/pull/136/commits/869ba884c5e4af129d90c7b4a7c2e5045efc62ec

File: README.md [hunk 4]
```diff
@@ -45,3 +38,11 @@ at Function.Module._load (node:internal/modules/cjs/loader:828:14)
 ```
 then please try to downgrade to node 15 or node 14 (seems to be linked with
 https://github.com/electron-userland/electron-builder/issues/5668).
+
+## Legacy generators
+
+To generate legacy projects (not recommended), please run the following command:
+```shell
+npm install generator-symphony@1.7.7
```
Comment (symphony-youri) [inline README.md:46]: let's maybe detail that legacy projects are .net, node and legacy python/legacy java ones?

We would have to update https://docs.developers.symphony.com/developer-tools/developer-tools/symphony-bot-generator#quick-start
and remove https://docs.developers.symphony.com/building-bots-on-symphony/configuration/configure-your-bot-for-bdk  I would say

this is going to be outdated too https://docs.developers.symphony.com/building-bots-on-symphony/tutorials/sdk/build-an-interactive-bot-using-the-sdk

**Review by symphony-elias — APPROVED** [review]
LGTM !

**Pull request merged**